documentation/src/main/asciidoc/topics/con_xsite_backup_strategies.adoc

```diff
@@ -1,5 +1,41 @@
-[id='xsite_backup_strategies-{context}']
+[id='backup-strategies_{context}']
 = Backup Strategies
-{brandname} clusters can use different strategies for backing up data to remote sites.
 
-{brandname} replicates across sites at the same time that writes to local caches occur. For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+{brandname} replicates data between clusters at the same time that writes to caches occur.
+For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+
+To back up data to a different cluster, {brandname} can use either a synchronous or asynchronous strategy.
+
+[discrete]
+== Synchronous Backups
+When {brandname} replicates data to backup locations, it waits until the operation completes before writing to the local cache.
+
+You can control how {brandname} handles writes to the local cache if backup operations fail.
+For example, you can configure {brandname} to attempt to abort local writes and throw exceptions if backups to remote sites fail.
+
+Synchronous backups also support two-phase commits with caches that participate in optimistic transactions.
+The first phase of the backup acquires a lock.
+The second phase commits the modification.
+
+[IMPORTANT]
+====
+Two-phase commit with cross-site replication has a significant performance impact because it requires two round-trips across the network.
+====
+
+[discrete]
+== Asynchronous Backups
+When {brandname} replicates data to backup locations, it does not wait until the operation completes before writing to the local cache.
+
+Asynchronous backup operations and writes to the local cache are independent of each other.
+If backup operations fail, write operations to the local cache continue and no exceptions occur.
```

Comment:
probably worth mention the backup operation is retried in case of failure until the remote site disconnects.